hw_8(3).py

```diff
@@ -0,0 +1,79 @@
+#1
+import re
+EMAIL_REGEX = re.compile( r"[A-Za-z0-9-_]+(.[A-Za-z0-9-_]+)*@""[A-Za-z0-9-]+(.[A-Za-z0-9]+)*(.[A-Za-z]{2,})")
```

Comment:
http://codingismycraft.com/index.php/2017/10/04/adding-descriptors-to-your-python-arsenal/?doing_wp_cron=1593468511.2042360305786132812500